main.js

```diff
@@ -15,8 +21,25 @@ function onArrowKeyClicks(event) {
   } else if (event.code === 'ArrowRight') {
     $car.className = 'car face-east';
     data.direction = 'east';
-  } else {
+  } else if (event.code === 'ArrowLeft') {
     $car.className = 'car face-west';
     data.direction = 'west';
   }
 }
+
+function moveCarRight() {
+  var leftVal = parseInt($car.style.left);
+  leftVal += 5;
+  $car.style.left = leftVal + 'px';
+  data.location.x = $car.style.left;
+  data.location.y = $car.sytle.top;
+}
+
+function startCar(event) {
+  if (event.code !== 'Space') {
+    return;
+  }
+  setInterval(moveCarRight, 16);
+}
```

Comment:
Instead of making a separate function for this, you should move this to be a part of your existing `keydown` event handler.